roles/openstack-setup/tasks/users.yml

```diff
@@ -15,8 +15,8 @@
                     login_password={{ secrets.admin_password }}
                     insecure={{ insecure|default(omit) }}
   with_items: "{{ keystone.services }}"
-  when: 
-    - (item.name.enabled is undefined or item.name.enabled|bool)
+  when:
+    - ({{item.name}}.enabled is undefined or {{item.name}}.enabled|bool)
```

Comment:
ah yeah, my bad, didn't see the default filter earlier. I can make this change in a PR i'm working now that touches this code. Good catch @chengtcli  .